### ISSUE TITLE
UI: High: Fixing Messages

### ISSUE THREAD

**Issue by ChetanaKoulagi** [LetsTagOn/LetsTagOnThoughtClan#1]
1. If an email id is registered already, it gives error "oops something went wrong". It should just say this email is already registered.
2. Other message updates sent as an attachment over email.

**Comment by Viswa-Vinod** [comment]
"oops something went wrong" is the title of the modal (the error message box). The actual error message will be in the body of the modal. The title of the modal should be left generic. So I changed it to 'Registration Error'. I believe what you are asking for - 'email is already registered' - should appear in the body of the modal. Please let me know what you are seeing in the body of the message. 

As for the rest of the messaging changes sent through email, points 2,3 and 4 - are backend java changes. Backend engineer should handle that. I have fixed 1, 5 and 6. I have pushed all my changes to vinod/featurebranch.

**Comment by ChetanaKoulagi** [comment]
Yes, you are right on the 'email is already registered' should appear in the body of the modal.

For others, will ask to be fixed at backend.

**Comment by Viswa-Vinod** [comment]
Does it not appear now? What appears in the body of the modal now?

**Comment by Viswa-Vinod** [comment]
According to the code, the modal simply displays whatever message is received from the backend in the body of the modal. So, if you want the body of the modal to have the right message, you may want to check with the backend engineer.

**Comment by ChetanaKoulagi** [comment]
Ok. Will check at backend.

**Comment by sathyab28** [comment]
Shilpa's backend changes has been pushed
